Services/Curl.php

```diff
@@ -95,7 +95,9 @@ public function sendRequest($url, $method, array $options = array(), $payload =
         $curlMetaData = (object) curl_getinfo($this->curl);
 
         $this->curlOptionsHandler->reset();
-        function_exists(curl_reset($this->curl)) ? curl_reset($this->curl) : curl_setopt_array($this->curl, $this->curlOptionsHandler->getOptions());
+        if (function_exists('curl_reset')) {
```

Comment:
sorry, you are right.
I lost previous line 
>>$this->curlOptionsHandler->reset();
here is the end of the day in Ukraine =)
please check it.